packaging/rpm/flightctl-agent.spec

```
@@ -2,7 +2,7 @@
 
 Name: flightctl-agent
 Version: 0.0.1
-Release: 2%{?dist}
+Release: 2.20240313105953187195.fix_data_dir%{?dist}
```

```suggestion
Release: 3%{?dist}
```

To match the changelog, I guess the other stuff is from packit.